Aries/Aries/Admin/Proofs/Messages/PresentationRequestApproveMessage.swift

```diff
@@ -13,18 +13,18 @@ public struct PresentationRequestApproveMessage: BaseOutboundAdminMessage {
     public let id: String
     public let transport: TransportDecorator
     public let presentationExchangeId: String
-    public let selfAttestedAttributes: [String : String]
+    public let selfAttestedAttributes: [String : String]?
```

Comment:
```suggestion
    public let selfAttestedAttributes = [String : String])()
```
This defaulting to an empty object matches more with consistency of AMAA, my bad.

Reply:
Fixed!